tools/Voron.Recovery/CommandLineApp.cs

```diff
@@ -164,6 +164,15 @@ private static void ConfigureRecoveryCommand()
 
                         config.IgnoreDataIntegrityErrorsOfAlreadySyncedTransactions = ignoreDataIntegrityErrorsOfAlreadySyncedTransactions;
                     }
+                    
+                    if (ignoreInvalidPagesInARawArg.HasValue())
+                    {
+                        var value = ignoreInvalidPagesInARawArg.Value();
+                        if (bool.TryParse(value, out var ignoreInvalidPagesInARaw) == false)
+                            return ExitWithError($"{nameof(config.IgnoreDataIntegrityErrorsOfAlreadySyncedTransactions)} argument value ({value}) is invalid", cmd);
```

Comment:
Invalid argument name in the message - `{nameof(config.IgnoreDataIntegrityErrorsOfAlreadySyncedTransactions)}`